src/calendar-app/calendar/gui/eventpopup/EventPreviewView.ts

```
@@ -202,6 +208,29 @@ export class EventPreviewView implements Component<EventPreviewViewAttrs> {
 		if (sanitizedDescription == null || sanitizedDescription.length === 0) return null
 		return this.renderRow(Icons.AlignLeft, [m.trust(sanitizedDescription)], true)
 	}
+
+	private renderCalendar(calendarName: string | undefined, calendarColor: string | undefined, calendarRenderType: TranslationKey | undefined) {
```

Any specific reason for using ```string | undefined``` instead of ```?: string``` ?

---

I'll raise error instead of returning null/undefined. This values should never be undefined.